library/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="chucker_color_primary">#81d4fa</color>
```

Comment:
Same here, having the night palette from material.io would be beneficial :)

Reply:
You mean the link to it? Will add, of course.
Except using colors from material I also tried to respect original Chucker colors, like green cursor in search or links for empty states :)

Reply:
Same thing as in my comment for `day` colors - palette still has old colors naming.
Here it is if I put primary color from `night` - https://material.io/resources/color/#!/?view.left=0&view.right=1&primary.color=01579b&secondary.color=00e676&secondary.text.color=000000
I took colors from this image https://storage.googleapis.com/spec-host-backup/mio-design%2Fassets%2F1PZrjuQLwNHJ1ZyWVEI7Kp_z6_fAOSR33%2Fdarktheme-adjustedcolor-baselines-color-scheme-comparison.png
which is placed here https://material.io/design/color/dark-theme.html#ui-application